### ISSUE TITLE
Add tracking for both React NativeScript and Svelte Native projects

### ISSUE THREAD

**Issue by rdlauer** [NativeScript/nativescript-cli#4947]
We should start tracking usage of the new community-driven [React NativeScript](https://github.com/shirakaba/react-nativescript) and [Svelte Native](https://svelte-native.technology/) initiatives in the CLI.

**Comment by etabakov** [comment]
Let's track those as "React" and "Svelte" respectively.

**Comment by rosen-vladimirov** [comment]
Acceptance Criteria:
- [ ] Projects created as described in https://github.com/shirakaba/react-nativescript#setup should be tracked as "React"
- [ ] Projects created as described in https://svelte-native.technology/should be tracked as "Svelte"
- [ ] JS or TS projects (without nativescript-angular or nativescript-vue in them) should be tracked as `React` after `npm install react-nativescript --save` is executed.
- [ ] JS or TS projects (without nativescript-angular or nativescript-vue in them) should be tracked as `Svelte` after `npm install svelte-native --save` is executed.
- [ ] Project which is tracked as React or Svelte should be available for `tns update` command, i.e. when you execute the `tns update` in such project, it should pass and update your `tns-core-modules`, runtimes and `nativescript-dev-webpack` versions.

> NOTE: You can check how CLI tracks each project by checking the cd2 value in the analyticsLogFile, i.e.:
```
tns usage-reporting enable
tns prepare <platform> --analyticsLogFile analyticsData.txt
```
After command exits check the analyticsData.txt file and search for:
```
Setting custom dimension cd2 to value Svelte
```
or 
```
Setting custom dimension cd2 to value React
```

**Comment by shirakaba** [comment]
Not sure if this is the best place for it, but either way, here's a starter template for React NativeScript. The commit history shows how I built it following `tns create mycoolapp --tsc`.

It follows the steps of https://github.com/shirakaba/react-nativescript#setup exactly, except for the fact that I've pinned the installed dependency versions.

https://github.com/shirakaba/react-nativescript-starter
